Interface.C

```diff
@@ -104,9 +113,12 @@ void preciceAdapter::Interface::configureMesh(const fvMesh& mesh)
         for (uint j = 0; j < patchIDs_.size(); j++)
         {
             // Get the face centers of the current patch
-            const vectorField faceCenters =
+            vectorField faceCenters =
                 mesh.boundaryMesh()[patchIDs_.at(j)].faceCentres();
 
+            if (cellDisplacement != nullptr && resetDisplacement_)
+                faceCenters -= cellDisplacement->boundaryField()[patchIDs_.at(j)];
```

Comment:
A comment for the intention is missing. Not only for relating this to restarting, but also to explain the mechanism.

I would expect something like (if I understand correctly):

```suggestion
            // Move the interface according to the current values of the cellDisplacement field,
            // to account for any displacements accumulated before restarting the simulation.
            // This is information that OpenFOAM reads from its result/restart files.
            // If the simulation is not restarted, the displacement should be zero and this line should have no effect.
            if (cellDisplacement != nullptr && resetDisplacement_)
                faceCenters -= cellDisplacement->boundaryField()[patchIDs_.at(j)];
```